tests/php/TestRegistration.php

```
@@ -0,0 +1,189 @@
+		// Verify patterns have the correct category
+		foreach ($expected_patterns as $pattern_name) {
+			$pattern = $patterns_registry->get_registered($pattern_name);
+			if ($pattern && isset($pattern['categories'])) {
+				$this->assertContains(
+					'lsx-tour-operator',
+					$pattern['categories'],
+					"Pattern '{$pattern_name}' should be in 'lsx-tour-operator' category"
+				);
+			}
+		}
+	}
```

PHPMD (2.15.0)

The method test_patterns_registered is not named in camelCase. (undefined)

(CamelCaseMethodName)